recipes/kraken2/meta.yaml

```diff
@@ -1,19 +1,19 @@
 {% set name = "kraken2" %}
-{% set version = "2.0.7_beta" %}
-{% set sha256 = "baa160f5aef73327e1a79e6d1c54b64b2fcdaee0be31b456f7bc411d1897a744" %}
+{% set version = "2.0.8_beta" %}
+{% set sha256 = "f2a91fc57a40b3e87df8ac2ea7c0ff1060cc9295c95de417ee53249ee3f7ad8e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/DerrickWood/kraken2/archive/v2.0.7-beta.tar.gz
+  url: https://github.com/DerrickWood/kraken2/archive/v2.0.8-beta.tar.gz
   sha256: {{ sha256 }}
   patches:
     - Makefile.patch
 
 build:
-  number: 3
+  number: 1
```

Comment:
Should the build number be set to `0`?
https://bioconda.github.io/guidelines.html#build-numbers

Reply:
Typo!